robottelo/cli/factory.py

```diff
@@ -391,6 +392,46 @@ def make_gpg_key(options=None):
     return create_object(GPGKey, args, options)
 
 
+@cacheable
+def make_content_credential(options=None):
+    """Creates a content credential.
+
+    In Satellite 6.8 only gpg_key is supported.
+
+    :param options: Check options using `hammer content-credentials create --help` on satellite.
+
+    :returns ContentCredential object
+    """
+    # Organization ID is a required field.
+    if not options or not options.get('organization-id'):
+        raise CLIFactoryError('Please provide a valid ORG ID.')
+
+    # Create a fake gpg key file if none was provided
+    if not options.get('key'):
+        (_, key_filename) = mkstemp(text=True)
```

Comment:
May be a personal preference thing, but the `()` are not necessary when unpacking, `_, key_filename = <expr>` would be valid.

Reply:
OK. change made. Thank you for the review.